### PR TITLE
Login page design.

### PR DESCRIPTION
**Pull request by Rishika-S-V**
Added login page design.
Files includes figma file, a pdf file assets and a mainfest.yaml file.

### CODE REVIEW

File: website/wireframes/login/mainfest.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+fonts:
+    Title: "Roboto Slab",
+    Login: "Radley",
+    Roles_name: "Roboto Slab",
+    Username: "Sanchez",
+    Password: "Sanchez",
+    remember_me: "Saira",
+    login_button: "Francois One"
+
```
Comment (naveeng2402) [inline website/wireframes/login/mainfest.yaml:9]: Are all the fonts available in google fonts?

Comment (Rishika-S-V) [reply website/wireframes/login/mainfest.yaml:9]: Yes i have checked it

**Review by Hari-7822 — COMMENTED** [review]
fine... 
but we can use **Roles_Border** as a replacement to **Roles_circle**;

File: website/wireframes/login/mainfest.yaml (hunk 1 above)
Comment (jkokilaCSE) [reply website/wireframes/login/mainfest.yaml:9]: yes I have checked it

**Review by Hari-7822 — COMMENTED** [review]
Is there any svg format for images(student, teacher, parent). cause its in png format.

**Pull request closed without merging**